addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -1,50 +1,55 @@
 # binding
 binding.kodi.name = Kodi Binding
-binding.kodi.description = Das Kodi Binding erlaubt das Kodi Media Center zu steuern 
+binding.kodi.description = Dieses Binding integriert das Kodi Media Center. Durch diese kann das Media Center gesteuert werden.
 
 # thing types
 thing-type.kodi.kodi.label = Kodi Media Center
-thing-type.kodi.kodi.description = Kodi Media Center Binding
-thing-type.config.kodi.kodi.ipAddress.label = Netzwerkaddresse
-thing-type.config.kodi.kodi.ipAddress.description = IP Addresse oder Host Name des Kodi Media Centers
+thing-type.kodi.kodi.description = Kodi Media Center.
+
+# thing types config
+thing-typ.property.kodi.kodi.version = Version
+
+thing-type.config.kodi.kodi.ipAddress.label = IP Adresse
+thing-type.config.kodi.kodi.ipAddress.description = Lokale IP Adresse oder Hostname des Kodi Media Centers.
 thing-type.config.kodi.kodi.port.label = Port
-thing-type.config.kodi.kodi.port.description = Port des Web Socket Services des Kodi Media Centers
+thing-type.config.kodi.kodi.port.description = Port des Kodi Media Centers.
```

Comment:
@CWempe Thanks for review. It is written down in the documentation which port must be given (including default value). We maybe have to clarify if PR #2304 is merged.